site/styles/_meta-detail.scss

```diff
@@ -576,6 +584,33 @@
   }
 }
 
+.element-switcher-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+
+  .sponsor {
+    display: none;
+    @media (min-width: 1200px) {
+      display: flex;
+    }
+    padding-top: 10px;
+    margin-left: 50px;
+  }
+}
+
+.sponsor {
+  display: flex;
```

Comment:
If you change the sponsor text span to a div, you can remove `display: flex` and leave it on default (block).

Reply:
Oh yep I see
- flex and flex-column removed
- span changed to div